docs/Manual/04 Fusing.md

```diff
@@ -116,55 +116,57 @@ for i in range(0, 16, 4):
             for jj in range(4):
                 C[i+ii, j+jj] *= B[i+ii, j+jj]
 ```
+## Constraints of Fusing Dimension
+The fusing dimension comes with certain constraints that are discussed from the `safety` perspective with examples. 
 
 ### Constraint 1: the fusing dimension is executed sequentially
-The fusing dimension has a special constraint, which does not apply to other dimensions. Specifically, the fusing dimension cannot be parallelized, vectorized, or tensorized (see [Section 7](<07%20Plans%20-%20Vectorization%20and%20Parallelization.md>) ) and it must be executed sequentially. This constraint enables the safety guarantee discussed below.
+Unlike other dimensions that allow parallelization and vectorization (parallelization and vectorization are presented in [Section 7](<07%20Plans%20-%20Vectorization%20and%20Parallelization.md>) ), the fusing dimension can neither be parallelized nor vectorized. It has an inherent execution constraint and must be executed sequentially. However, this sequential execution constraint allows the safety guarantee discussed below.   
 
 ### Safety
-The fused schedule (before applying any subsequent transformations) is always logically equivalent to executing the original schedules one-by-one. However, is it safe? Recall that a schedule is considered safe if its underlying logic is guaranteed not to change, regardless of how we transform it. The safety of a fully fused schedule depends on the circumstances:
+Before applying any subsequent transformations, the fused schedule is always logically equal to executing the original schedules sequentially. However, is it safe? Recall that a schedule is considered safe if the underlying logic is guaranteed to be unchanged regardless of the applied transformation. Nonetheless, the safety of the fused schedule depends on the circumstances under which it is transformed. Consider the below discussed cases to understand how the safety of a fully fused schedule works: 
```

Comment:
I think we can drop the "Nonetheless, " here (for brevity)

Reply:
This paragraph became more verbose than necessary..., I think we can remove the final sentence and still keep the meaning intact.

"transformation. The safety of a fused schedule depends on circumstances that may break logic equivalence:"